packages/paste-website/stories/Pricing.stories.tsx

```diff
@@ -0,0 +1,378 @@
+import { Avatar } from "@twilio-paste/avatar";
+import { Badge } from "@twilio-paste/badge";
+import { Box } from "@twilio-paste/box";
+import { Button } from "@twilio-paste/button";
+import { Card } from "@twilio-paste/card";
+import { DetailText } from "@twilio-paste/detail-text";
+import { DisplayHeading } from "@twilio-paste/display-heading";
+import { Heading } from "@twilio-paste/heading";
+import { BusinessIcon } from "@twilio-paste/icons/esm/BusinessIcon";
+import { SelectedIcon } from "@twilio-paste/icons/esm/SelectedIcon";
+import { UserIcon } from "@twilio-paste/icons/esm/UserIcon";
+import type { GenericIconProps } from "@twilio-paste/icons/esm/types";
+import { Paragraph } from "@twilio-paste/paragraph";
+import { Separator } from "@twilio-paste/separator";
+import { TBody, THead, Table, Td, Th, Tr } from "@twilio-paste/table";
+import { VisualPickerRadio, VisualPickerRadioGroup } from "@twilio-paste/visual-picker";
+import * as React from "react";
+
+export default {
+  title: "Website/PricingExamples",
+};
+
+const PricingCard = ({
```

Comment:
praise: love the reusability of this *chef's kiss*

Reply:
question: Since we're shipping this as a pattern, is this component/composition straightforward enough for a consumer to just copy/paste into their own feature?

Reply:
Follow up from standup: @nkrantz will take a look too ^ but if we have the plain layout example in the code example on the doc site, I'm good with this.